### PR TITLE
Agency Shifts | See information

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx [hunk 1]
```diff
@@ -132,8 +132,12 @@ function getDateRequestValue(date: moment.Moment, time: string): string {
   return `${date.format('YYYY-MM-DDT')}${time}:00`;
 }
 
-function getOptionListValue(list: SFPeopleOption[]): string[] {
-  return list.map((o: SFPeopleOption) => o.asyncObject.id);
+function getOptionListValue(list: SFPeopleOption[]): ShiftMember[] {
+  return list.map((o: SFPeopleOption) => ({
+    id: o.asyncObject.id,
+    name: o.name,
+    avatar_thumbnail_url: o.avatarUrl
```
Comment (martinweingart) [inline src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx:139]: This `props` are not needed.

File: src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx [hunk 2]
```diff
@@ -156,8 +160,14 @@ function getShiftRequestValue(value: ShiftFormValue): ShiftRequest {
...
     },
     areas: getOptionListValue(value.areas),
-    members: getOptionListValue(value.members),
-    supervisor: value.supervisor?.asyncObject.id
+    participants: getOptionListValue(value.participants),
+    supervisor: value.supervisor
+      ? {
+          id: value.supervisor.asyncObject.id,
+          name: value.supervisor.name,
+          avatar_thumbnail_url: value.supervisor.avatarUrl
```
Comment (martinweingart) [inline src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx:168]: This `props` are not needed.

**Review by martinweingart — CHANGES_REQUESTED** [review]

File: src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx]: done

File: src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx (hunk 2 above)
Comment (Yopo-Nicolas) [reply src/Views/AgencyShifts/ShiftFormModal/ShiftFormModal.tsx]: done

**Review by martinweingart — APPROVED** [review]
LGTM

**Pull request merged**